swift/ios-shoppe-demo/ServiceManagers/FullStoryManager.swift

```diff
@@ -9,12 +9,15 @@
 import Foundation
 import FullStory
 
-/**
+/*
  FullStory Manager is a file full of functions and enums that can make the FullStory API easier to apply to areas of your project.
+    - Identifying a user
```

Comment:
Keep a consistent list (i.e. if one item starts with a capital, apply that to the rest).  If it's helpful, link to the FullStory feature you're referencing.  Like a `(see https://help.fullstory.com/hc/en-us/articles/360044349073-FullStory-Private-by-Default#exclude_mask_unmask)`. If it makes more sense to link to external doc closer to one of your convenience functions, feel free to.